### ISSUE TITLE
Having problems running after recent update

### ISSUE THREAD

**Issue by idoerg** [tseemann/prokka#291]
Hi Torsten,

I am getting the following when trying to run the latest & greatest from the github repo. (Prokka worked fine before I updated). Running on Ubuntu 16.04.

```
$ ./bin/prokka --setupdb
Time::Piece object version 1.29 does not match bootstrap parameter 1.20_01 at /usr/lib/x86_64-linux-gnu/perl/5.22/DynaLoader.pm line 210.
Compilation failed in require at ./bin/prokka line 25.
BEGIN failed--compilation aborted at ./bin/prokka line 25.

```

**Comment by tseemann** [comment]
@idoerg yes, it seems i have a clash. does this fix it?
```
cd /path/to/prokka
rm -fr perl5
mkdir perl5
```

**Comment by tseemann** [comment]
Fixed by d6adb2c79ceedfdfa621eaec19ce25e656e51c5a
